bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -265,25 +285,73 @@ message SwapExactInResponse {
 message WithdrawQuotesRequest {
   // base_asset: The base asset against which the accumulated quotes that the user wants to withdraw. e.g. USDT pool
   string base_asset = 1;
-}
-
-// WithdrawQuotesResponse: Response to the WithdrawQuotesRequest. Contains information about the withdrawn quotes.
+  // deposit_address: The address where withdrawn quotes should be deposit. If not provided, the default sender address
+  // would be used. Note it can be overwritten for specific quotes.
+  optional string deposit_address = 2;
+  // quotes: Details of how specific quotes should be withdrawn. Note that quotes that are not valid for this base, or
+  // were not collected would not cause an error - they will be simply ignored.
+  repeated QuoteWithdrawal quotes = 3;
```

Comment:
Since we are already providing a deposit_address, isn't it enough to just send the asset list?
```suggestion
  repeated string quotes = 3;
```

Reply:
If someone wants to send different quote assets to different address, they could just call this endpoint again with different deposit address. I prefer to keep the requests as simple as possible.

Reply:
I was thinking of it. The reason I went this way is basically reducing request amount (to further reducing traffic avoiding extra proto/app-level headers). But I don't mind going either way.